.github/workflows/deploy_to_rubygems.yml

```diff
@@ -0,0 +1,22 @@
+name: Deploy to RubyGems
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy-to-rubygems:
+    name: Deploy to RubyGems
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      id-token: write
+    steps:
+      # Set up
+      - uses: actions/checkout@v4
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+          ruby-version: "3.2"
```

Comment:
does it matter which version of ruby we use here?

Reply:
I don't think so? At best, we should use the same version between this "validation" workflow and the actual workflow used for the "release" of the documentation itself, but I think we can change that only if we find issues with it? WDYT?

So I just picked one of the newest and most stable versions

Reply:
I don't think so, so I just picked one of the newest and most stable versions